src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/data/SupplierStatementRepositoryTest.java

```diff
@@ -23,9 +26,16 @@
 @SpringBootTest
 public class SupplierStatementRepositoryTest {
 
-    @Autowired SupplierStatementRepository repo;
-    @Autowired ObjectMapper objectMapper;
-    @Autowired ClockProvider clockProvider;
+    @Autowired private SupplierStatementRepository repo;
+    @Autowired private ClockProvider clockProvider;
```

Comment:
is this change needed (adding `private`)?

Reply:
not needed. just to be consistent.